src/rpc/server.cpp

```diff
@@ -257,7 +257,7 @@ static const CRPCCommand vRPCCommands[] =
 CRPCTable::CRPCTable()
 {
     unsigned int vcidx;
-    for (vcidx = 0; vcidx < (sizeof(vRPCCommands) / sizeof(vRPCCommands[0])); vcidx++)
+    for (vcidx = 0; vcidx < ARRAYLEN(vRPCCommands); vcidx++)
```

Comment:
The idx isn't needed, so a C++11 range-based for loop would be better suited.

See conflicting pull #19528